docker/init/03-users.js

```diff
@@ -1,75 +1,25 @@
-const users = db.getSiblingDB('trello-clone')
-
 const user1 = {
   _id: ObjectId('6230ef5c91ecd051dc75df37'),
-  username: 'Marek Kowalski',
+  username: 'Marek Kowalski2137',
   passwordHash: 'pass123hash1',
   avatarId: 23,
-  dashboards: [
-    {
-      _id: ObjectId('6230ef67edfb0cbac5e5a246'),
-      imageCoverId: 'photo-1640622842924-fb0017b9d786',
-      title: 'New dashboard 1',
-      role: 'ADMIN',
-      users: [
-        {
-          _id: ObjectId('6230ef6cd3594bb0cc6ed836'),
-          avatarId: 23
-        }
-      ]
-    },
-    {
-      _id: ObjectId('6230ef70b0df36f40ca5c7fa'),
-      imageCoverId: 'photo-1640622842924-fb0017b9d788',
-      title: 'New dashboard 2'
-    }
-  ]
+  memberOfDashboards: [ObjectId('6230ef67edfb0cbac5e5a246'), ObjectId('6230ef70b0df36f40ca5c7fa')]
 }
 
 const user2 = {
   _id: ObjectId('6230ef75c1a96ed7fbb4d3c0'),
   username: 'Jan Przykładowy',
   passwordHash: 'pass123hash',
   avatarId: 2,
-  dashboards: [
```

Comment:
what?! why did you remove? :o

Reply:
https://github.com/kacper-cyra/camp-trello-clone/pull/112#discussion_r873225690